### PR TITLE
non-interactive: add missing udev rules

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: nix/zfs-minimal.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  zfs = pkgs.zfs.override {
+    # this overrides saves 10MB
+    samba = pkgs.coreutils;
+  };
+in
+{
+  services.udev.packages = [ zfs ]; # to hook zvol naming, etc.
+  # unsure if need this, but in future udev rules could potentially point to systemd services.
+  systemd.packages = [ zfs ];
+  environment.defaultPackages = lib.mkForce [ zfs ]; # this merges with out noninteractive module.
```
Comment (nbdd0121) [inline nix/zfs-minimal.nix:13]:
```suggestion
  environment.defaultPackages = lib.mkForce [ zfs ]; # this merges with outer noninteractive module.
```

**Review by nbdd0121 — APPROVED** [review]
This was fast! Thank you!

LGTM, just a nit.

**Comment by Mic92** [comment]
@mergify squash

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**